### PR TITLE
chore: image comparison replaced with other verifications

### PR DESCRIPTION
**Pull request by Valentina1133**
As was discussed with @anastasiyaig, I replaced images comparison with other verifications and deleted steps with user images comparison (as it's not possible to get source path for such cases) 

The only place I left image comparison is here - https://github.com/status-im/desktop-qa-automation/blob/master/gui/screens/community.py#L44, @StateOf-Vlado can you please do it? (when I tried to get icon's properties color.name and emoji instead of grabbing image - it gave me an error I never understood)

### CODE REVIEW

File: tests/communities/test_communities.py [hunk 1]
```diff
@@ -26,15 +25,10 @@ def test_create_community(user_account, main_screen: MainWindow, params):
         community_screen = create_community_form.create(params)
 
     with step('Verify community parameters in community overview'):
-        with step('Icon is correct'):
-            community_icon = main_screen.left_panel.get_community_logo(params['name'])
-            image.compare(community_icon, 'button_logo.png', timout_sec=5)
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:31]: Let's skip these steps without deleting them, I will fix it later.

File: tests/communities/test_communities.py [hunk 2]
```diff
@@ -26,15 +25,10 @@ def test_create_community(user_account, main_screen: MainWindow, params):
...
         with step('Name is correct'):
             assert community_screen.left_panel.name == params['name']
         with step('Members count is correct'):
             assert '1' in community_screen.left_panel.members
-        with step('Logo is correct'):
-            image.compare(community_screen.left_panel.logo, 'logo.png')
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:37]: Let's skip these steps without deleting them, I will fix it later.

File: tests/communities/test_communities.py [hunk 3]
```diff
@@ -54,7 +48,6 @@ def test_create_community(user_account, main_screen: MainWindow, params):
         assert community.name == params['name']
         assert community.description == params['description']
         # assert '1' in community.members TODO: Test on linux, members label is not visible
-        image.compare(community.image, 'logo_in_settings.png')
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:57]: Let's skip these steps without deleting them, I will fix it later.

File: tests/communities/test_communities.py [hunk 4]
```diff
@@ -127,13 +120,8 @@ def test_edit_community(main_screen: MainWindow, params):
 
     with step('Verify community parameters in community screen'):
         community_setting.left_panel.back_to_community()
-        with step('Icon is correct'):
-            community_icon = main_screen.left_panel.get_community_logo(params['name'])
-            image.compare(community_icon, 'button_updated_logo.png')
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:132]: Let's skip these steps without deleting them, I will fix it later.

File: tests/communities/test_communities.py [hunk 5]
```diff
@@ -127,13 +120,8 @@ def test_edit_community(main_screen: MainWindow, params):
...
         with step('Name is correct'):
             assert community_screen.left_panel.name == params['name']
-        with step('Logo is correct'):
-            image.compare(community_screen.left_panel.logo, 'updated_logo.png')
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:136]: Let's skip these steps without deleting them, I will fix it later.

File: tests/communities/test_communities.py [hunk 6]
```diff
@@ -142,7 +130,6 @@ def test_edit_community(main_screen: MainWindow, params):
         assert community_info.name == params['name']
         assert community_info.description == params['description']
         assert '1' in community_info.members
-        image.compare(community_info.image, 'logo_in_settings_updated.png')
```
Comment (StateOf-Vlado) [inline tests/communities/test_communities.py:145]: Let's skip these steps without deleting them, I will fix it later.

**Review by StateOf-Vlado — APPROVED** [review]
Let's skip the deleted steps; I will fix it later.

File: tests/communities/test_communities.py (hunk 6 above)
Comment (Valentina1133) [reply tests/communities/test_communities.py:145]: Ok. I commented these steps and left link to a separate task - to replace in these steps and also https://github.com/status-im/desktop-qa-automation/blob/master/gui/screens/community.py#L44 - here. I will assign it to you then

Comment (anastasiyaig) [reply tests/communities/test_communities.py:145]: i suggest to wait a bit with this task. I dont see any strong pros for testing through comparing images right now, there is other stuff to cover. @StateOf-Vlado we still can have the task and the code, but implementation should be deferred for now.

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**